### PR TITLE
Changing indent width to 4 

### PR DESCRIPTION
**Pull request by GreatRSingh**
# Changing indent width to 4

## Description

Fix #3159

Changed indent in 14 files.## Checklist

- [x] My code follows [the style guidelines of this project](https://deepchem.readthedocs.io/en/latest/development_guide/coding.html)
  - [x] Run `yapf -i <modified file>` and check no errors (**yapf version must be  0.32.0**)
  - [x] I have performed a self-review of my own code
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: deepchem/feat/molecule_featurizers/mol2vec_fingerprint.py [hunk 1]
```diff
@@ -11,187 +11,187 @@
...
+      Calc6ulates ECFP (Morgan fingerprint) and returns identifiers of substructures as 'sentence' (string).
+      Returns a tuple with 1) a list with sentence for each radius and 2) a sentence with identifiers from all radii
+      combined.
+      NOTE: Words are ALWAYS reordered according to atom order in the input mol object.
+      NOTE: Due to the way how Morgan FPs are generated, number of identifiers at each radius is smaller
+
+      Parameters
+      ----------
+      mol : rdkit.Chem.rdchem.Mol
+      radius : float
```
Comment (gusty1g) [inline deepchem/feat/molecule_featurizers/mol2vec_fingerprint.py:24]: The docstring should be formatted to follow numpy docstring style [conventions](https://numpydoc.readthedocs.io/en/latest/format.html). You can use the script [here](https://gist.github.com/gusty1g/3ed5abd147d44bca65383c4d992bb7f5) to format docstring.

Comment (GreatRSingh) [reply deepchem/feat/molecule_featurizers/mol2vec_fingerprint.py:24]: That script [https://gist.github.com/gusty1g/3ed5abd147d44bca65383c4d992bb7f5](here) is not working properly in indenting the docstring. What should I do in this case?

Comment (gusty1g) [reply deepchem/feat/molecule_featurizers/mol2vec_fingerprint.py:24]: That's an issue we are also fixing - that was a quick code which I wrote some time back. We can try fixing the script or fixing the docstring manually.

**Comment by GreatRSingh** [comment]
Changes in:
atomic_coordinates.py
bp_symmetry_function_input.py
circular_fingerprint.py
coulomb_matrices.py

did manual changes to match numpy docstring indentation rules

**Comment by GreatRSingh** [comment]
@gusty1g ready for review

**Review by gusty1g — APPROVED** [review]

**Pull request merged**